draft-ietf-quic-recovery.md

```diff
@@ -837,37 +837,43 @@ approximately equivalent to two TLPs before an RTO in TCP.
 This duration is computed as follows:
 
 ~~~
-(smoothed_rtt + max(4 * rttvar, kGranularity) + max_ack_delay) *
-    kPersistentCongestionThreshold
+PTO * kPersistentCongestionThreshold
 ~~~
 
-For example, assume:
+where PTO is computed as described in {{computing-pto}}.
```

Comment:
If we are to (re)use PTO in the equation, I think it would be better to clarify that max_ack_delay should be used regardless of the packet number space for which a loss is newly observed.

That's what we do in the existing text, and IMO that aligns with the fact that persistent congestion is a connection-wide event that happens across all the packet number spaces.

Reply:
That's a good point, and not an obvious one.

Reply:
That's a really good point. I've brought back the expanded expression, since it is perhaps less confusing, and I've added a sentence to point out this difference as well.